### ISSUE TITLE
Equation 3 in the paper

### ISSUE THREAD

**Issue by CBY-9527** [LittlePey/SFD#11]
Equation 3 in the paper shows that the loss includes rpn, iou, aux1, aux2, and depth completion loss. It seems that the depth completion network and the detection network are trained together in an end-to-end manner. However, the provided codes only have the detection network, and directly use the obtained pseudo point clouds and the point clouds to train the detection network. The code is trained in stages, and there seems to be a deviation from Equation 3.

**Comment by RG2806** [comment]
Equation 3 in the paper shows that the loss includes rpn, iou, aux1, aux2, and depth completion loss. It seems that the depth completion network and the detection network are trained together in an end-to-end manner. However, the provided codes only have the detection network, and directly use the obtained pseudo point clouds and the point clouds to train the detection network. The code is trained in stages, and there seems to be a deviation from Equation 3.

is the interpretation correct of the previous comment?

**Comment by LittlePey** [comment]
The model can be trained end to end. For convenience, we use the pre-trained depth completion model to generate pseudo clouds.

**Comment by RG2806** [comment]
thanks, for the clarification
